src/components/StartExploringPage.jsx

```diff
@@ -44,6 +44,18 @@ const StartExploringPage = ({ t, embed, runTour = false, joyrideCallback }) => (
           <li>
             <ExploreSuggestion name='XKCD Archives' cid='QmdmQXB2mzChmMeKY47C43LxUdg1NDJ5MWcKMKxDu7RgQm' type='dag-pb' />
           </li>
+          <li>
+            <ExploreSuggestion name='Distributed Wikipedia mirror' cid='bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze' type='hamt-sharded-directory' />
```

Comment:
When user clicks on this dag, filenames will look weird, let's make it clear why we show this example (huge DAG, special case  of dag-pb + UnixFS):

```suggestion
            <ExploreSuggestion name='HAMT-sharded Wikipedia mirror (>20M files)' cid='bafybeiaysi4s6lnjev27ln5icwm6tueaw2vdykrtjkwiphwekaywqhcjze' type='hamt-sharded-directory' />
```


>  ![2023-05-26_13-39](https://github.com/ipfs/ipld-explorer-components/assets/157609/27e48979-74b0-476a-8a2a-cf4f29325a0b)